resources/Microsoft.Windows.Setting.Accessibility/Microsoft.Windows.Setting.Accessibility.psm1

```diff
@@ -249,8 +250,10 @@ class VisualEffect
     # Key required. Do not set.
     [DscProperty(Key)] [string] $SID
     [DscProperty()] [bool] $AlwaysShowScrollbars = $false
+    [DscProperty()] [bool] $EnableMonoAudio = $false
 
     hidden [string] $DynamicScrollbarsProperty = 'DynamicScrollbars'
+    hidden [string] $EnableMonoAudioProperty = 'AccessibilityMonoMixState'

```

Comment:
Instead of defining these values as hidden instance properties, recommend defining as hidden static properties:

```suggestion
    static hidden [string] $DynamicScrollbarsProperty = 'DynamicScrollbars'
    static hidden [string] $EnableMonoAudioProperty = 'AccessibilityMonoMixState'
```

Then they can be accessed in the code as `[VisualEffect]::EnableMonoAudioProperty`.

Thinking about this from both a semantic perspective (these are functionally constants you want to access, not properties of the resource instance itself) and from a serialization perspective (JSON serialization in PowerShell includes hidden properties without an easy way to tell it otherwise, which can be very confusing for users--especially in v3), it's best to avoid using hidden instance properties for resources like this.

Another option would be to define the values as a variable or in a non-resource class that only has static properties, like `[SettingAccessibilityConstants]` or similar.

Reply:
Should the same be done for the other hidden variables above?